plugins/woocommerce-blocks/assets/js/blocks/active-filters/test/block.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import type { BlockAttributes } from '@wordpress/blocks';
+import '@testing-library/jest-dom';
+import { fireEvent, screen, within } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	initializeEditor,
+	selectBlock,
+} from '../../../../../tests/integration/helpers/integration-test-editor';
+import '../';
```

Comment:
Is it necessary?

Reply:
It imports and registers the block we're testing, so it's necessary to have it insertable within the headless editor.